### ISSUE TITLE
Fiches especes - Liens automatiques vers d'autres ressources

### ISSUE THREAD

**Issue by camillemonchicourt** [PnEcrins/GeoNature-atlas#83]
De la même manière qu'un lien est automatiquement créé vers l'INPN sur chaque fiche especes, proposer d'autres liens de ce genre. 
Exemple : http://www.pifh.fr/pifh/pifh/index.php/fiche_descriptive/OuvrirFicheDescriptive/104419-0
Contrainte : Pouvoir regénérer dynamiquement l'URL avec le CD_ref
A voir : si certains liens ne sont proposés que pour certains rangs taxonomiques
A réfléchir : Que cette génération de liens soient génériques car ils peuvent différer d'un atlas à un autre. Donc les gérer dans une table de la BDD.

**Comment by gildeluermoz** [comment]
Je ne pense pas qu'il y ai des ressources en lignes aussi exhaustives que celle de l'inpn.
L'automatisation par l'application me semble difficilement envisageable. Je vois plutôt une génération "oneshoot" de medias de type URL pour un groupe taxonomique donné. Ceci lorsque l'url permet d'exploiter le cd_nom ou le cd_ref.

**Comment by camillemonchicourt** [comment]
Manuellement c'est déjà possible.

Médias - Lien Web - URL

**Comment by DonovanMaillard** [comment]
c'est pas gérable des liens faits manuellement vers d'autres sites "partenaires" sélectionnés par les thématiciens par exemple ou les plus spécialistes de tel ou tel taxon? 

Par exemple renvoyer vers les sites de lafranchis ou lepinet pour les pap, renvoyer sur tela-orthoptera pour les criquets, vers tela-botanica pour Cédric etc? Ca augmente encore un peu le travail en réseau mais ça permet de valoriser les sites fiables...

**Comment by gildeluermoz** [comment]
Oui, je le ferai moi, en sql. Avec votre aval et relecture du résultat, Sur un groupe complet.
Faisabilité et pertinence à creuser ensemble à l'occas.

**Comment by DonovanMaillard** [comment]
oui et j'imagine que ca devrait passer par une convention, ou au moins un accord avec le site ciblé?

**Comment by camillemonchicourt** [comment]
Pas de convention ou d'accord pour faire un lien vers un autre site. 
Pour moi il faut une table générique pour gérer ça et dans celle-ci on peut indiquer le(s) rangs taxonomiques ou groupe(s) concernés. 
Et en effet il faut que l'on puisse recréer dynamiquement les liens vers les pages donc qu'elles se basent sur un CD-ref comme le font l'INPN ou GeoNature-atlas. 

A garder au chaud pour des versions futures.

**Comment by Adrien-Pajot** [comment]
Lien aussi avec des ressources biblio, c'est pas mal demandé pour COBIODIV. On le met dans le projet V2.
